### PR TITLE
chore(auth): migrate lib to lib-v1

### PR DESCRIPTION
**Pull request by israx**
#### Description of changes:
Migrates auth content in lib to lib-v1
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/directory/directory.mjs [hunk 1]
```diff
@@ -1229,27 +1234,37 @@ export const directory = {
           {
             title: 'Social sign-in (OAuth)',
             route: '/lib-v1/auth/social',
-            filters: ['android', 'ios', 'flutter', 'js']
+            filters: ['android', 'ios', 'flutter']
```
Comment (Samaritan1011001) [inline src/directory/directory.mjs:1237]: Why is js removed here?

File: src/directory/directory.mjs [hunk 2]
```diff
@@ -1306,6 +1326,11 @@ export const directory = {
             route: '/lib-v1/auth/delete_user',
             filters: ['android', 'ios', 'flutter']
           },
+          {
```
Comment (Samaritan1011001) [inline src/directory/directory.mjs:1329]: This has the same path as the previous block with the same title, should the path be different or have only one or the other?

**Review by Samaritan1011001 — APPROVED** [review]

File: src/directory/directory.mjs (hunk 1 above)
Comment (israx) [reply src/directory/directory.mjs:1237]: we are changing it to another path for js and rn

File: src/directory/directory.mjs (hunk 2 above)
Comment (israx) [reply src/directory/directory.mjs:1329]: yeah so currently `lib` changed the title to `Delete user account` but kept the same path, so basically I'm changing the title to `lib-v1` and keeping the same path too

**Pull request merged**